File: fizzbuzz.js
const fizzBuzz = () => {
    for( let i = 0; i < 101; i++) {
        if( i % 3 === 0 && i % 5 === 0) {
            i++
            console.log("FizzBuzz")
        }
        else if( i % 3 === 0 ) {
            i++
            console.log("Fizz")
        }
        else if( i % 5 === 0 ) {
            i++
            console.log("Buzz")
        }
        console.log(i)
    }
}

fizzBuzz();

File: fizzbuzz.rb
def fizzbuzz
    count = 0
    while count < 100 do 
        count += 1
        if count % 15 == 0
            count += 1
            puts "FizzBuzz"
        elsif count % 3 == 0
            count += 1
            puts "Fizz"
        elsif count % 5 == 0
            count += 1
            puts "Buzz"
        end
        puts count
    end
end

fizzbuzz